build.gradle

```diff
@@ -1,36 +1,17 @@
-buildscript {
-    ext {
-        springBootVersion = '2.5.1'
-    }
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
-    }
-}
 plugins {
-    id "io.freefair.lombok" version "5.3.0"
+    id 'java'
+    id 'jacoco'
+    id 'org.springframework.boot' version '2.6.1'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
 }
-apply plugin: 'java'
-apply plugin: 'jacoco'
-apply plugin: 'idea'
-apply plugin: 'org.springframework.boot'
-apply plugin: 'io.spring.dependency-management'
 
 group 'uk.gov.digital.ho.hocs'
 version '1.0-SNAPSHOT'
 sourceCompatibility = '11'
+
 repositories {
     mavenCentral()
-    mavenLocal()
```

Comment:
Should these be removed?

Reply:
I think so, there's no mavenLocal on other projects and nothing will be on drone- we should be consistent.
As for artifactory I really don't like the idea of needing a username and password to build a project... 
I don't understand why the schema is its own project at all - where is the potential for reuse(or why we have a schema!)

Reply:
happy to defer, of course

Reply:
The mavenLocal is so you can build the schema and push it locally so locally you don’t need the credentials. 

For it being on artifactory, it’s there because it is consumed by the HOF team. It is also what the public form is validated against and should be private to prevent misuse.

Reply:
I'll put these back, but the repo is public https://github.com/UKHomeOffice/hocs-ukvi-complaint-schema so there is little point publishing to artifactory...

Reply:
Good point, we can review this as part of the case-creator partial re-write.